courses/machine_learning/deepdive/08_image/mnist_linear.ipynb

```diff
@@ -9,9 +9,7 @@
    "source": [
```

Comment:
<p>I'd add the following line here to make sure we are all using the same TF 2.0 version:</p><p> </p><p>!pip freeze | grep tf-nightly-2.0-preview || pip install tf-nightly-2.0-preview</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Hmm, when I run it, the notebook is still using the beta version that comes with the Deep Learning VM. I'm a little nervous about changing the version that comes with the notebook as I've seen that result in some weird bugs. I added a line asking the user to check that they're running TF2 in case they aren't using a Deep Learning VM, and to run the above line if not. Thoughts?</p>

Reply:
<p>At some point, I believe we needed to have the tf-nightly installed in order to deploy the model. If the default TF 2.0 works with deployment, we should use that, and replace the version by the default version in the line above. </p>

Reply:
Good points. I just figured out how to use docker today to deploy TF2 to ai-platform. I'll try the notebook version of TF2 (2.0.0-beta1) with it and see if it works. :)